translation/source/faq.xml

```diff
@@ -32,8 +32,8 @@
 	<string name="gameplay">Gameplay</string>
 	<string name="howBulletBlitzEtcDecided">How are Bullet, Blitz and other time controls decided?</string>
 	<string name="basedOnGameDuration">Lishogi time controls are based on estimated game duration = %1$s
-For instance, the estimated duration of a 5+3 game is 5 × 60 + 40 × 3 = 420 seconds.</string>
-	<string name="durationFormula">(clock initial time) + 40 × (clock increment)</string>
+For instance, the estimated duration of a 5+3|4 game is 60 × 5 + 40 × 3 + 25 × 4 = 520 seconds.</string>
+	<string name="durationFormula">60 × (clock initial time in minutes) + 40 × (clock increment) + 25 × (clock byoyomi)</string>
```

Comment:
For this one it's fine to overwrite it, I think, because the old line didn't fully explain explain it. Just please change the 40 to 60, since we changed average game length from 40 moves to 60.